packages/devextreme/testing/tests/DevExpress.ui.widgets/popup.tests.js

```diff
@@ -169,6 +170,50 @@ QUnit.module('basic', () => {
         assert.strictEqual($overlayContent.attr('role'), 'dialog');
     });
 
+    QUnit.test('aria-labelledby is equal to title id', function(assert) {
+        const instance = $('#popup').dxPopup({
+            title: 'title',
+            visible: true,
+        }).dxPopup('instance');
+
+        const $overlayContent = instance.$content().parent();
+        const titleId = $overlayContent.find(`.${TOOLBAR_LABEL_CLASS}`).attr('id');
+
+        assert.strictEqual($overlayContent.attr('aria-labelledby'), titleId);
+    });
+
+    QUnit.test('aria-labelledby is not setted when title is null', function(assert) {
+        const instance = $('#popup').dxPopup({ visible: true }).dxPopup('instance');
+
+        const $overlayContent = instance.$content().parent();
+
+        assert.strictEqual($overlayContent.attr('aria-labelledby'), undefined);
+    });
+
+    QUnit.test('aria-labelledby is correct in runtime change', function(assert) {
+        const instance = $('#popup').dxPopup({
+            title: 'title',
+            visible: true,
+        }).dxPopup('instance');
+
+        const $overlayContent = instance.$content().parent();
+        const titleIdFirst = $overlayContent.find(`.${TOOLBAR_LABEL_CLASS}`).attr('id');
+
+        assert.strictEqual($overlayContent.attr('aria-labelledby'), titleIdFirst);
+
+        instance.option('title', 'title 2');
+        const titleIdSecond = $overlayContent.find(`.${TOOLBAR_LABEL_CLASS}`).attr('id');
+
+        assert.strictEqual($overlayContent.attr('aria-labelledby'), titleIdSecond);
+        assert.notStrictEqual($overlayContent.attr('aria-labelledby'), titleIdFirst);
+
```

Comment:
[minor] just a sug
```suggestion
        const secondTitle = 'title2';
        instance.option('title', secondTitle);
        assert.strictEqual($overlayContent.attr('aria-labelledby'), secondTitle);

```
i think its more clear and and we don't need notStrictEqual check

Reply:
aria-labelledby should be ID of the label element. Not a simple string.